Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM node:8.5.0-alpine
+
+MAINTAINER Mr. David Sibobbabley
```

Comment:
should be Dockta

Reply:
Dr. Dockta David Sibobabbley Jr. Esq. IV MD